static/js/components/ProfileProgressControls.js

```diff
@@ -40,6 +42,10 @@ export default class ProfileProgressControls extends React.Component {
       if (programIdForEnrollment && addProgramEnrollment) {
         addProgramEnrollment(programIdForEnrollment);
       }
+
+      if ( isLastTab ) {
+        sendGAEvent('profile-form', 'completed', SETTINGS.user.username);
```

Comment:
Could we ever reach this point as an anonymous user? i.e. where `SETTINGS.user` will be `null`?